lib/util/device-types.ts

```diff
@@ -9,14 +11,24 @@ const find = <T>(array: T[], predicate: (el: T) => boolean) => {
 	}
 };
 
-const includes = <T>(array: T[], el: T) => array && array.indexOf(el) >= 0;
+const includes = <T>(array: T[] | undefined, el: T) =>
+	array != null && array.indexOf(el) >= 0;
```

Comment:
I will refactor this at a later point of time, along with splitting all util imports and exposing some of them.